<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KoShadowConfigWidget</class>
 <widget class="QWidget" name="KoShadowConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>98</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" columnstretch="0,0,1" name="gridLayout_3">
   <item column="0" row="0">
    <widget class="QCheckBox" name="shadowVisible">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item column="1" row="0" rowspan="2">
    <widget class="QDial" name="shadowAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchTarget">
      <double>3.700000000000000</double>
     </property>
    </widget>
   </item>
   <item column="2" row="0">
    <layout class="QGridLayout" columnstretch="0,0,1" name="gridLayout_2">
     <item column="0" row="0">
      <widget class="QToolButton" name="shadowColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Blur:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="2" row="0">
      <widget class="KisDoubleParseUnitSpinBox" name="shadowBlur"/>
     </item>
    </layout>
   </item>
   <item column="2" row="1">
    <layout class="QGridLayout" columnstretch="0,1" name="gridLayout">
     <item column="0" row="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Distance:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="KisDoubleParseUnitSpinBox" name="shadowOffset"/>
     </item>
    </layout>
   </item>
   <item column="1" row="2">
    <widget class="QWidget" name="SpecialSpacer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleParseUnitSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>kis_double_parse_unit_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>